samples/samples/snippets.py

```diff
@@ -91,13 +91,13 @@ def update_instance(instance_id):
             labels={
                 "sample_name": "snippets-update_instance-explicit",
             },
-            edition=spanner_instance_admin.Instance.Edition.STANDARD,  # Optional
+            edition=spanner_instance_admin.Instance.Edition.ENTERPRISE,  # Optional
```

Comment:
In order to merge split API sample I had to change this to STANDARD in that PR so that the tests pass.
Reverting back that change in this PR.